workflows/rnaseq-ref-index/build-index.nf

```diff
@@ -34,12 +36,40 @@ process salmon_index{
     """
 }
 
+process salmon_index_full_sa{
+  container 'quay.io/biocontainers/salmon:1.3.0--hf69c8f4_0'
+  publishDir "${params.ref_dir}/salmon_index", mode: 'copy'
+  // try dynamic memory (28.GB so 2x will fit in r4.2xlarge)
+  memory { 28.GB * task.attempt}
+  cpus 8
+  errorStrategy { task.exitStatus in 137..140 ? 'retry' : 'terminate' }
+  maxRetries 1
+  input:
+    tuple val(index_base), path(reference), val(kmer)
+    path genome
+  output:
+    path "${index_base}_k${kmer}_full_sa"
+  script:
+    """
+    gunzip -c ${genome} \
+      |grep "^>" | cut -d " " -f 1 \
+      |sed -e 's/>//g' > decoys.txt
+    cat ${reference} ${genome} > gentrome.fa.gz
```

Comment:
I've never built one of these before but these steps look consistent with [the tutorial](https://combine-lab.github.io/alevin-tutorial/2019/selective-alignment/).